airbyte-integrations/connectors/destination-mssql-v2/src/main/kotlin/io/airbyte/integrations/destination/mssql/v2/config/MSSQLSpecification.kt

```diff
@@ -47,35 +47,68 @@ class MSSQLSpecification : ConfigurationSpecification() {
     @get:JsonSchemaInject(json = """{"examples":["public"],"default":"public","order":3}""")
     val schema: String = "public"
 
+    @get:JsonSchemaTitle("Authentication Method")
+    @get:JsonPropertyDescription("The authentication method which is used to access the database.")
+    @get:JsonProperty("authentication_method")
+    @get:JsonSchemaInject(json = """{"order":4}""")
+    val authenticationMethod: AuthenticationMethod = SqlPassword()
```

Comment:
Can we add a test for this? I remember having to change to `lateinit var` for SSL methods because it would fail to load and always default the value set here (`SqlPassword` in this case).